sender-sqs/src/main/java/zipkin/reporter/sqs/SQSSender.java

```diff
@@ -36,23 +36,23 @@
 import static zipkin.internal.Util.checkNotNull;
 
 /**
- * Zipkin Sender implementation that sends spans to an SQS queue. This uses the
- * AmazonSQSBufferedAsyncClient which batches messages to improve throughput and lower API requests
- * to SQS. By default this client will send messages every 200ms or when ever at most 10 messages
- * are buffered. This doesn't change the Zipkin Reporter batching but instead augments it to
- * lower API requests to SQS.
+ * Zipkin Sender implementation that sends spans to an SQS queue.
  *
- * This sends (usually TBinaryProtocol big-endian) encoded spans to an SQS queue.
+ * <p>The {@link AsyncReporter} batches spans into a single message to improve throughput and lower
+ * API requests to SQS. Based on current service capabilities, a message will contain roughly 256KiB
+ * of spans.
+ *
+ * <p>This sends (usually TBinaryProtocol big-endian) encoded spans to an SQS queue.
  */
 @AutoValue
-public abstract class AwsBufferedSqsSender extends LazyCloseable<AmazonSQSBufferedAsyncClient> implements Sender {
+public abstract class SQSSender extends LazyCloseable<AmazonSQSAsyncClient> implements Sender {
```

Comment:
renamed to the simpler SQS naming, as Aws isn't important (especially as there are SQS clones)